tanzu-build-service/install-airgapped.hbs.md

```diff
@@ -25,6 +25,14 @@ Approximately 10&nbsp;GB of registry space is required when using the `full` dep
 
 - Your Docker registry must be accessible with user name and password credentials.
 
+## <a id='DeprecatedFeatures'></a> Deprecated Features
+
+* Users are encouraged to use the SBOM format published in
+  [Buildpacks API 0.7.0](https://buildpacks.io/docs/reference/spec/migration/buildpack-api-0.6-0.7/#new-standardized-sbom-format).
+  Older (legacy) SBOM formats are deprecated and are disabled by default in TBS. Support for legacy SBOM formats be
```

Comment:
same comment as here: https://github.com/pivotal-cf/docs-build-service/pull/307#discussion_r972295910

Reply:
Thanks @tylerphelan, I reverted this PR and opened a new draft PR here with this change: https://github.com/pivotal/docs-tap/pull/1570.